### DISCUSSION TITLE
Database condition LIKE with escaping method

### DISCUSSION THREAD

**Discussion by tes123id** [phalcon/cphalcon#15705] [Q&A]
Is there a way to crreate a query with LIKE condition using Query Builder.
Using the conventional method, might result the query break, because the variable have to be escaped.

**Comment by quasipickle** [comment]
`LIKE` can be treated like any other `WHERE` condition, with one caveat.  The "?" needs to be in the value of the variable you're binding - it can't be in the query.  This isn't a Phalcon problem - it's how PDO works.

So this won't work:
```php
->andWhere(
    "myField LIKE '?:foo:'",
    [
        "foo"=>$foo
    ]
)
```
This will:
```php
->andWhere(
    "myField LIKE :foo:",
    [
        "foo"=>"?".$foo
    ]
)
```
